client/js/src/client.ts

```diff
@@ -1129,9 +1129,16 @@ async function check_space_status(
 
 			setTimeout(() => {
 				check_space_status(id, type, status_callback);
-			}, 1000);
-			break;
-		// poll for status
+			}, 1000);  // poll for status
+			break; 
+		case "PAUSED":
+			status_callback({
+				status: "paused",
+				load_status: "pending",
+				message: "This space has been paused.",
```

Comment:
Maybe we could prompt the viewer to duplicate the space in this case (since the author has probably paused it for cost reason).

```suggestion
				message: "This space has been paused by the author. If you would like to try this demo consider duplicating the space",
```

Maybe we should add a link here but we don't have a great way of including links atm.